data/brands/amenity/payment_centre.json

```diff
@@ -71,6 +71,20 @@
         "name:bg": "Фастпей"
       }
     },
+    {
+      "displayName": "Fina - Financijska agencija",
+      "locationSet": {"include": ["hr"]},
+      "tags": {
+        "brand": "Financijska agencija",
+        "brand:wikidata": "Q12631015",
+        "name": "Fina",
+        "amenity": "payment_centre",
+        "bureau_de_change": "yes",
+        "money_transfer": "western_union",
+        "office": "financial",
```

Comment:
I'm not fond of things getting multiple top level tags, in this case `amenity` and `office`. Are you sure this is correct tagging?

Reply:
Yes, it seems most precise... While most citizens will use them for paying bills (thus `amenity=payment_centre`), they do offer more advanced financial services to SMEs too. 

But if you think it is problematic, I can remove `office=financial`?

Reply:
I would not have added `office=financial`, however it's not a hard rule, NSI does have entries with 2 categories, eg FedEx and Chuck e cheese.

I'll leave it for other people to provide opinions.

Reply:
I would also be in favour of removing `office=financial`. Besides, OSM wiki [claims](https://wiki.openstreetmap.org/wiki/Tag:office%3Dfinancial_advisor#See_also) `office=financial` should be used for offices not open to the general public.

Reply:
Thanks for feedback, I've removed  `office=financial` now.